examples/playbooks/54.upgrade-etcd.yaml

```
@@ -0,0 +1,27 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+- name: Upgrade etcd
+  hosts: master,etcd
+  serial: 1
+  become: true
+  vars:
+    etcd_address: "{{ ansible_host }}"
+    upgrade: True
```

ansible-lint suggests using `true` all in lower-case
```suggestion
    upgrade: true
```

https://ansible.readthedocs.io/projects/lint/rules/yaml/